custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1250,99 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+def is_launched_awc_incentive_report(performance_row, awcagg_row):
```

Comment:
This method name is misleading. Starting with `is_` implies that it would be a boolean check, but if successful it returns a list of four things. I would recommend renaming it to be more clear, e.g. `get_row_data_if_awc_launch_data_is_wrong` (not sure that's right but hopefully the idea is clear)

Reply:
ok..will do

Reply:
changed